var/spack/repos/builtin/packages/py-werkzeug/package.py

```diff
@@ -9,9 +9,10 @@
 class PyWerkzeug(PythonPackage):
     """The Swiss Army knife of Python web development"""
 
-    homepage = "http://werkzeug.pocoo.org"
+    homepage = "https://werkzeug.palletsprojects.com/en/2.2.x/"
```

Comment:
```suggestion
    homepage = "https://werkzeug.palletsprojects.com"
```
If you remove the version from the URL it will automatically update for the next release